### ISSUE TITLE
Map out what fields can be displayed to proponent from 'Project Overview'

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/cas-cif#1406]
#### Describe the task

As we build out the CIF external web app, we need to understand if there are any fields that can't/shouldn't be displayed to the operator. This ticket is to understand the internal + external application fields from Project Overview.

#### Acceptance Criteria

- [x] map out/document all fields in application
- [x] review with CIF team to determine what fields can be made viewable to proponents
- [x] update map/document with feedback from CIF team
- [x] update our team

#### Additional context

- Nan may already have a spreadsheet with fields that can be repurposed
- Review with CIF team might be able to be done async

**Comment by LindsayMacfarlane** [comment]
FYI @suhafa - I've created this ticket.

**Comment by suhafa** [comment]
@LindsayMacfarlane this ticket is good to be closed after your review! Some learnings summarized (also updated the 'Clarifying Questions for CIF team' document): 
- We decided to rename Facility Name to 'Project Facility.' There is some confusion here from the Ops team as to whether they want to include this field to begin with, so we've decided to put it on hold for now
- We've gotten a better idea as to the Ops team's naming conventions for RFP
- We learnt that we need to include project submission date and time in the app
- We learnt the 'Project Name' convention the Ops team uses. They append the last three numbers of the RFP number as a prefix to the Project Name
 - We learnt that the team finds it fairer to not show draft projects on the internal side until the proponent submits the projects
- We were able to clarify that the Internal team does not want to populate the app with more fields from 'Project Overview' on the internal side as they would like to avoid the user from repeating information they input in the templates. 
- We were able to confirm where the links in the footer and sub-header should point to. 
- They don't see a use for allowing the user to see which project manager was assigned the project to.
